getting_started/build-guide/build-from-scratch.rst

```diff
@@ -224,47 +247,48 @@ Clone the header repository.
    cd newlib-xtensa
    git checkout -b xtensa origin/xtensa
 
-Build and install the headers for each platform.
+Build and install for each platform.
 
 .. code-block:: bash
 
-   #Baytrail/Cherrytrail
-   ./configure --target=xtensa-byt-elf --prefix=/home/$USER/work/sof/xtensa-root
+   XTENSA_ROOT=/home/$USER/work/sof/xtensa-root
```

Comment:
Why not just `$HOME`?

Reply:
Yes, better. Thanks for the thorough review!